packages/opentelemetry-node/docs/intro.asciidoc

```diff
@@ -0,0 +1,52 @@
+////
+Goal of this doc:
+Provide all the information a user needs to determine if the product is a good enough fit for their use case to merit further exploration
+
+Assumptions we're comfortable making about the reader:
+* They are familiar with Elastic
+* They are familiar with OpenTelemetry
+////
+
+[[intro]]
+== Elastic Distribution for OpenTelemetry Node.js
+
+++++
+<titleabbrev>Introduction</titleabbrev>
+++++
+
+:language: Node.js
+:distro_name: Elastic Distribution for OpenTelemetry {language}
+
+include::release-status.asciidoc[]
+
+////
+✅ Intro
+////
+The {distro_name} ("the distro") is a {language} package that provides:
+
+* An easy way to instrument your application with OpenTelemetry.
+* Configuration defaults for best usage.
+
+////
+✅ What is it?
+✅ Why use it?
+////
+A _distribution_ is a wrapper around an upstream OpenTelemetry repository with some customizations. The {distro_name} is a wrapper around the https://opentelemetry.io/docs/languages/js[OpenTelemetry SDK for {language}] and includes the following customizations:
+
+* Uses Elastic-specific processors that ensure optimal compatibility when exporting OpenTelemetry signal data to an Elastic backend like Elastic APM server.
+* Preconfigures the collection of tracing and metrics signals, applying some opinionated defaults, such as which sources are collected by default.
+* Ensures that the OpenTelemetry protocol (OTLP) exporter is enabled by default.
+
+NOTE: For more details about OpenTelemetry distributions in general, visit the https://opentelemetry.io/docs/concepts/distributions[OpenTelemetry documentation].
+
+////
+✅ How to use it?
+////
+Use the distro to start the OpenTelemetry SDK with your {language} application to automatically capture tracing data, performance metrics, and logs. Traces, metrics, and logs are sent to any OTLP collector you choose.
+
+Start with helpful defaults to begin collecting and exporting OpenTelemetry signals quickly. Then, further refine how you use the distro using extension methods that allow you to fully control the creation of the underlying tracer and metric providers.
```

Comment:
Is this language from one of the other languages distros? I'm curious to compare.

Reply:
The "How to use it?" paragraphs mostly come from [your existing docs](https://github.com/elastic/elastic-otel-node/blob/main/packages/opentelemetry-node/docs/landing-page.mdx?plain=1#L21-L25) mixed with some language about defaults adapted from the .NET docs.

If this paragraph is not accurate for Node.js we can remove or refine the language. If there's a better way to describe a _start with defaults, refine as needed_ approach, we can refine for all distro docs.

Reply:
Can we drop this paragraph entirely?
"using extension methods" isn't really how the Node.js SDK is configured.
Do [whatever is appropriate for this language's SDK] "to fully control the creation of the underlying tracer and metric providers" is okay, but feels like a slightly obtuse way to say that the SDK/distro is configurable, because it gets into the weeds of tracer and metric "providers" that is something one only really needs to know about if getting more familiar with OTel API/SDK design.